modules/ROOT/pages/use-environment-properties-on-tests-cookbook.adoc

```diff
@@ -0,0 +1,70 @@
+= Testing with Environment Properties
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Assume that you have the following flow that obtains data from an external API and you need to consume it from a test environment when executing your MUnit tests:
+
+[source,xml]
+----
+<http:request-config name="HTTP_Request_configuration">
+	<http:request-connection host="sampleapi.com/api/v1/" port="80"/>
+</http:request-config>
+<flow name="requestApiFlow">
+	<http:request method="GET" config-ref="HTTP_Request_configuration" path="/employee"/>
+</flow>
+----
+
+To consume data from a QA API endpoint when running your MUnit tests.
```

Comment:
Making an example of something that I see often that is visible here:

Why use the wording "consume it from a test environment" in the opening para and then reword it to "consume data from a QA API endpoint" when introducing the steps? 

I assume that these mean the same thing, but it would be easier on an audience to refer to this action in one boilerplate wording, to avoid any chance of confusion. 

Always try to pick the one most precise but simple phrase or term, and then use it (and only it) throughout your deliverable.